torch/csrc/autograd/profiler.h

```diff
@@ -42,7 +42,7 @@ inline int64_t getTime() {
   return duration_cast<nanoseconds>(clock::now().time_since_epoch()).count();
 #else
   // clock_gettime is *much* faster than std::chrono implementation on Linux
-  struct timespec t;
+  struct timespec t{};
```

Comment:
srsly xd what was wrong with that previously. This initialization really is a waste of time

Reply:
Idk about this case, I think it's because it's an old C struct or something